libs/SmartStore/src/com/salesforce/androidsdk/smartstore/store/SmartStore.java

```diff
@@ -1230,7 +1245,33 @@ public String getColumnType() {
             return columnType;
         }
     }
-    
+
+    /**
+      * Enum for type groups
+      */
+    public enum TypeGroup {
```

Comment:
In case we add more types in the future, this groupings should limit code change above.